tests/cases/test_ip_netns.sh

```diff
@@ -70,83 +70,5 @@ else
     exit 1
 fi
 
-
-
-# Exit with return value
```

Comment:
can we undo this big change here, i know it might be line numbering, but please change only the line that you need to change, and we can fix spacing and other stuff in another commit.

Reply:
its not just a few lines, the whole scrip is duplicated ;)
I reverted the change, we do cleanup separately as suggested.